### DISCUSSION TITLE
SB LERN 5.4 planning

### DISCUSSION THREAD

**Discussion by rhwarrier** [Sunbird-Lern/Community#120] [Announcements]
The following are the planned dates for SB Lern Release 5.4:

Closure of release scope and design discussions : 26 May 23
Sprint 1 : 29 May '23 - 09 Jun '23
Sprint 2 : 12 Jun '23 - 23 Jun '23
Testing, Regression & fixes : 26 Jun '23 - 06 Jul '23
Planned GA for SB Lern 5.4 : 07 Jul '23

The list of items that are to be taken up are being consolidated in the following [Jira filter](https://project-sunbird.atlassian.net/projects/LR/versions/10260/tab/release-report-all-issues) - these are items that focus on improving usability and efficiency of Lern components within the Sunbird suite of building blocks.

If there is a requirement for any additional items to be considered/ taken up, kindly call it out in the discussion thread so that it can be discussed by the community.

 

@mathewjpallan, @pkranga, @coolbung, @reshmi-nair
@SanthoshVasabhaktula, @vrayulu, @surendrasinghs, @pallakartheekreddy, @krgauraw, @gandham-santhosh, @kameshbhr, @AmiableAnil, @Jayaprakash8887, @anandp504, @maheshkumargangula
@surendrasinghs [Krishnaj20](https://github.com/Krishnaj20), [vinukumar-vs](https://github.com/vinukumar-vs)
@aishwaryashikshalokam @Prateek-slokam @aks30
@ashokreddy1208

**Comment by Namratakushwah** [comment]
Hello All,

Please find the Sunbird Lern Building block 5.4 sign-off here.

Environment: [[[Dev lern](https://dev.lern.sunbird.org/)]
Testing Status: Completed
Defects Dashboard: [[Link](https://project-sunbird.atlassian.net/browse/LR-611)]

Execution Sheet: [[Link](https://docs.google.com/spreadsheets/d/1VyVN-QNzbuvjdlEkTyHGOtw6-AFnTQFesn50jVLgXH4/edit#gid=0)]
Scope : [[Link](https://project-sunbird.atlassian.net/projects/LR/versions/10260/tab/release-report-all-issues)]
Building block sign-off template: [[Link](https://docs.google.com/spreadsheets/d/11DK55SwlyVDYnUTfOhay6Nj0doWOstkKBSH8qh8rPx0/edit#gid=0)]

cc: @rhwarrier @AmiableAnil  @chitranshu-keshav @ashokreddy1208 @PoornimaRajesh @vinukumar-vs @Krishnaj20
